### DISCUSSION TITLE
Why did `kitty` choose `golang` for `kittens`

### DISCUSSION THREAD

**Discussion by baseplate-admin** [kovidgoyal/kitty#6992] [General]
Hi,

I am a long time follower of `kitty` terminal project. In the previous iteration ( when i last took a look at sources about 2-3 years ago ),i didn't see any `golang` codes.

So my question is:
* Why did kitty chose `golang` to develop `kittens` ?

**Comment by kovidgoyal** [comment]
1) Single static executable easy to cross compile for all platforms
2) Reduced startup time
3) Decent stdlib and large number of community libraries (see go.mod for
which ones kitty uses)

**Comment by baseplate-admin** [comment]
Thanks for your answer

p.s : If you dont mind i would like to discuss about things.
